assets/src/components/template-inserter/edit.css

```diff
@@ -6,8 +6,7 @@
 	margin-right: 10px;
 }
 
-#amp-story-controls .editor-inserter__amp-inserter span:first-of-type,
-.amp-story-controls-reorder span:first-of-type {
```

Comment:
This `.amp-story-controls-reorder span:first-of-type` selector was first introduced in https://github.com/ampproject/amp-wp/commit/3c43aebf61df5e23b4eb666182441e36c9b44f82#diff-3052c4057908d4ffe0943926ac172b24R707

/cc @miina